src/twisted/internet/defer.py

```diff
@@ -159,6 +165,65 @@ def maybeDeferred(f, *args, **kw):
         return succeed(result)
 
 
+def unbox(deferred, raises=True, passthrough=True):
+    """
+    Unbox a L{Deferred} that has already been fully processed.
```

Comment:
Do other places use the lingo "fully processed", I'm not experienced enough to know, but I think common vernacular if that's what you mean is "that has already fired its result" or the like.